### PR TITLE
Store lint output in nextcloud

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
This one stores the master and the current lint html files in my cloud, so we can easily review what errors/warnings have increased.

### CODE REVIEW

File: scripts/lint/lint-up-wrapper.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+cd `dirname $0`
+ruby ./lint-up.rb $1 $2 $3
+
+if [ $? -eq 0 ]; then
+    echo "New master at: https://nextcloud.kaminsky.me/index.php/s/tXwtChzyqMj6I8v"
+    curl -u $4:$5 -X PUT https://nextcloud.kaminsky.me/remote.php/webdav/droneLogs/master.html --upload-file ../../build/reports/lint/lint.html
+else
+    echo "New results at https://nextcloud.kaminsky.me/index.php/s/tXwtChzyqMj6I8v ->" $6.html
+    curl -u $4:$5 -X PUT https://nextcloud.kaminsky.me/remote.php/webdav/droneLogs/$6.html --upload-file ../../build/reports/lint/lint.html
+fi
```
Comment (mario) [inline scripts/lint/lint-up-wrapper.sh]: new line...

File: scripts/lint/lint-up-wrapper.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+#1: GIT_USERNAME
+#2: GIT_TOKEN
+#3: BRANCH
+#4: LOG_USERNAME
+#5: LOG_PASSWORD
+#6: DRONE_BUILD_NUMBER
+
+cd `dirname $0`
```
Comment (mario) [inline scripts/lint/lint-up-wrapper.sh]: Fix this...

**Comment by AndyScherzinger** [comment]
👍

**Comment by mario** [comment]
👍

**Pull request merged**